NebulaAPI/DataStructures/ConcurrentPlayerCollection.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using NebulaAPI.GameState;
+using NebulaAPI.Networking;
+
+namespace NebulaAPI.DataStructures;
+
+public class ConcurrentPlayerCollection
+{
```

Comment:
I think the standard C# convention for private classes is to put at the bottom. It doesn't matter that much, but It did feel off when I was reading, though to be fair i don't use them often.

Though the convention does make sense to me. As with every purely style comment anyone ever leaves on a PR, I encourage you to feel absolutely no qualms about closing this comment and ignoring it.